.changeset/old-forks-scream.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/design-system-components": patch
```

Comment:
[Question] How is this not a breaking change? It feels not backwards-compat.

Reply:
Fair question. We're seeing it as a fix because it was meant to be `outage` (if you look at the guidance page on Helios)

Reply:
I guess I don't have any objection if no one has implemented it yet.

Reply:
No adoption yet. We're currently working on the first adoption in `cloud-ui` – that's how we identified the issue.